Makefile

```diff
@@ -224,6 +224,14 @@ data/out/country_extractions/worldpop1km: | data/out/country_extractions ## down
 	ls static_data/countries | parallel 'bash scripts/download_worldpop.sh {} 1km'
 	touch $@
 
+data/in/srtm: | data/in data/mid ## download srtm zipped tiles
```

Comment:
looks like we should have a separate target for folder creation and use it as dependency for "download srtm" target:

data/in/srtm: | data/in ##create folder
	mkdir -p $@

data/in/srtm/download_srtm: | data/in/srtm ## download srtm target
	....